datahub-web-react/src/app/entity/user/UserEditProfileModal.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { message, Button, Input, Modal, Typography, Form } from 'antd';
+import { useUpdateCorpUserPropertiesMutation } from '../../../graphql/user.generated';
+
+type Props = {
+    visible: boolean;
+    onClose: () => void;
+    onSave: () => void;
+    data: {
+        name: string | undefined;
+        title: string | undefined;
+        image: string | undefined;
+        team: string | undefined;
+        email: string | undefined;
+        slack: string | undefined;
+        phone: string | undefined;
+        urn: string | undefined;
+    };
+};
+/** Regex Validations */
+export const validUserName = new RegExp('^[a-zA-Z ]*$');
+
+export default function UserEditProfileModal({ visible, onClose, onSave, data }: Props) {
+    const [updateCorpUserPropertiesMutation] = useUpdateCorpUserPropertiesMutation();
+    const [form] = Form.useForm();
+
+    const [userName, setUserName] = useState<string | undefined>(data.name);
```

Comment:
Can we use a single state object to store all of these fields?

Reply:
Also, you don't need to prefix everything with 'user'. The fact that we are inside a UserEditProfileModal already implies that we are updating "user" fields